src/main/resources/application.properties

```diff
@@ -1,3 +1,6 @@
 kafka.broker.address=${KAFKA_BROKER_ADDR}
 kafka.group.name=${KAFKA_GROUP_NAME}
-kafka.consumer.topic=${KAFKA_CONSUMER_TOPIC}
\ No newline at end of file
+kafka.consumer.topic=${KAFKA_CONSUMER_TOPIC}
+kafka.retry.topic=${KAFKA_RETRY_TOPIC}
+kafka.producer.retries=${KAFKA_PRODUCER_RETRIES}
+chips.rest.endpoint=endpoint
```

Comment:
see previous comment, the endpoint will need to be retrieved from the message and placed on the end of a chips rest interfaces host string.

Reply:
Removed